test/common_plat/common/mask_common.c

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#include "config.h"
```

Comment:
@heyi-linaro no, this config.h is a part of build system and won't be included into distribution package or installed onto target system.